iothub/service/src/Common/Amqp/LegacyClientWebSocketTransport.cs

```diff
@@ -1,63 +1,52 @@
+using System.IO;
+using System.Net;
+using System.Net.WebSockets;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Common;
+
 namespace Microsoft.Azure.Amqp.Transport
 {
-    using System;
-    using System.IO;
-    using System.Net;
-    using System.Net.WebSockets;
-    using System.Threading.Tasks;
-    using Microsoft.Azure.Devices;
-    using Microsoft.Azure.Devices.Common;
-
-    sealed class LegacyClientWebSocketTransport : TransportBase
+    internal sealed class LegacyClientWebSocketTransport : TransportBase
     {
-        const string ClientWebSocketTransportReadBufferTooSmall = "LegacyClientWebSocketTransport Read Buffer too small.";
-        const int MaxReadBufferSize = 256 * 1024; // Max Read buffer size is hard-coded to 256k
-        static readonly AsyncCallback onWriteComplete = OnWriteComplete;
-
-        readonly IotHubClientWebSocket webSocket;
-        readonly EndPoint localEndPoint;
-        readonly EndPoint remoteEndPoint;
-        readonly TimeSpan operationTimeout;
-        readonly int asyncReadBufferSize;
-        readonly byte[] asyncReadBuffer;
-
-        bool disposed;
-        int asyncReadBufferOffset;
-        int remainingBytes;
-
-        public LegacyClientWebSocketTransport(IotHubClientWebSocket webSocket, TimeSpan operationTimeout,  EndPoint localEndpoint, EndPoint remoteEndpoint)
+        private const string ClientWebSocketTransportReadBufferTooSmall = "LegacyClientWebSocketTransport Read Buffer too small.";
+        private const int MaxReadBufferSize = 256 * 1024; // Max Read buffer size is hard-coded to 256k
+
+        private static readonly AsyncCallback s_onWriteComplete = OnWriteComplete;
+
+        private readonly IotHubClientWebSocket _webSocket;
+        private readonly EndPoint _localEndPoint;
+        private readonly EndPoint _remoteEndPoint;
+        private readonly TimeSpan _operationTimeout;
+        private readonly int _asyncReadBufferSize;
+        private readonly byte[] _asyncReadBuffer;
+
+        private bool _disposed;
+        private int _asyncReadBufferOffset;
+        private int _remainingBytes;
+
+        public LegacyClientWebSocketTransport(IotHubClientWebSocket webSocket, TimeSpan operationTimeout, EndPoint localEndpoint, EndPoint remoteEndpoint)
             : base("legacyclientwebsocket")
         {
-            this.webSocket = webSocket;
-            this.operationTimeout = operationTimeout;
-            this.localEndPoint = localEndpoint;
-            this.remoteEndPoint = remoteEndpoint;
-            this.asyncReadBufferSize = MaxReadBufferSize; // TODO: read from Config Settings
-            this.asyncReadBuffer = new byte[this.asyncReadBufferSize];
+            _webSocket = webSocket;
+            _operationTimeout = operationTimeout;
+            _localEndPoint = localEndpoint;
+            _remoteEndPoint = remoteEndpoint;
+            _asyncReadBufferSize = MaxReadBufferSize; // TODO: read from Config Settings
```

Comment:
Should we go ahead with reading from config settings?

Reply:
I don't know what is entailed in that TODO, and I wouldn't want to do it as part of simple code cleanup. @timtay-microsoft, do you have any context on this work item?